scripts/generator/generate.js

```diff
@@ -101,6 +101,10 @@ writer.writeSeparateJson(versions.react, hillaReactJsonTemplateFileName, hillaRe
 
 const hilla = process.env.HILLA || platform.replace(/^23/, 1);
 versions.core.hilla = {javaVersion: hilla};
+// write hilla version to hilla-react-versions.json as platform
+const hillaVersions = {platform : hilla};
```

Comment:
Could we also use `hillaVersions.platform` on the line for the regular `hilla` artifact above?

```
writer.writeSeparateJson(hillaVersions.platform, hillaJsonTemplateFileName, hillaJsonResultFileName, "platform");
```

Reply:
updated.